creation/interfaces-and-networks.md

```diff
@@ -548,64 +548,52 @@ the interface should be configured as follows.
 > **Note:** The network CIDR can be configured in the pod network
 > section using the `vmNetworkCIDR` attribute.
 
-#### masquerade - IPv6 support
-It is currently experimental, but `masquerade` mode can be used in IPv6 clusters
-(not dual stack - IPv6 only).
+#### masquerade - IPv4 and IPv6 dual-stack support
+
+It is currently experimental, but `masquerade` mode can be used in IPv4 and IPv6
+dual-stack clusters to provide a VM with an IP connectivity over both protocols.
 
 As with the IPv4 `masquerade` mode, the VM can be contacted using the pod's IP
-address - which will be an IPv6 one. Outgoing traffic is also "NAT'ed" to the
-pod's IPv6 address.
+address - which will be in this case two IP addresses, one IPv4 and one
+IPv6. Outgoing traffic is also "NAT'ed" to the pod's respective IP address
+from the given family.
 
-Unlike in IPv4, the configuration of the IP address, default route, and DNS is
+Unlike in IPv4, the configuration of the IPv6 address and the default route is
 not automatic; it should be configured via cloud init, as shown below:
 
-<pre>
-    kind: VM
-    metadata:
-      namespace: <i>your-favorite-namespace</i>
-    spec:
-      domain:
-        devices:
-          disks:
-            - disk:
-              bus: virtio
-              name: cloudinitdisk
-          interfaces:
-            - name: red
-              masquerade: {} # connect using masquerade mode
-              ports:
-                - port: 80 # allow incoming traffic on port 80 to get into the virtual machine
```

Comment:
do we need rng ? 
else sometimes there are problems with VM, because they dont have enough entropy